data/en/dotnettocftype.json

```diff
@@ -6,7 +6,7 @@
 	"related":[],
 	"description":"Explicitly converts a value returned by a .NET method to the corresponding ColdFusion data type.",
 	"params": [
-		{"name":"variableName","description":"Name of the .NET variable to convert.","required":true,"default":"","type":"Varible","values":[]}
+		{"name":"variableName","description":"Name of the .NET variable to convert.","required":true,"default":"","type":"Variable","values":[]}
```

Comment:
Hmm... should the type actually be "variableName", that is a valid CFML type?

Reply:
Given the example at https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-c-d/dotnettocftype.html I think `any` would be more appropriate since it's passing the .NET object and not the name of a variable.

Reply:
Yes you are correct @KamasamaK  , type **'any'** is more suitable in this case.